Issue #1485 Profile docs improvements

hferentschik opened the pull request:
- Adding output to the exmaple commands
- Expanding on the example workflow
- Clarify wording around active profile
- Making docs more compact by removing unnecessary fillers

kowen-rh commented on docs/source/using/profiles.adoc:
```diff
@@ -63,75 +69,66 @@ The default profile is *minishift*.
 
 [[example-workflows-profiles]]
-== Example Workflows
+== Example Workflow For Profile Configuration
```
"For" is not capitalized when using title case.

hferentschik replied:
right

kowen-rh commented on docs/source/using/profiles.adoc:
```diff
@@ -63,75 +69,66 @@ The default profile is *minishift*.
+== Example Workflow For Profile Configuration
 
-To create a new profile with specific configurations, run:
+To create a new profile and configure its the xref:../using/basic-usage.adoc#persistent-configuration[persistent configuration] you have two options.
```
Typo: "its the"

I'd also move the clause "you have two options" to the beginning of the sentence:

"You have two options to create a new profile and configure its...."

hferentschik replied:
+1 for re-arranging

kowen-rh commented on docs/source/using/profiles.adoc:
```diff
@@ -15,46 +15,52 @@ A profile is an instance of the Minishift VM along with all of its configuration
 
 Each {project} profile is created with its own configuration (memory, CPU, disk size, add-ons, and so on) and completely independent of other profiles.
-See xref:../using/basic-usage.adoc#environment-variables[environment variables], if you want to apply some common configurations (CPU, memory etc) to all profiles.
+Refer to the use of xref:../using/basic-usage.adoc#environment-variables[environment variables], if you want to make sure that certain configuration settings, for example `cpus` or `memory`, get applied to all profiles.
```
I like this rewrite. Just need to remove the comma after "environment variables".

hferentschik replied:
ok

kowen-rh commented on docs/source/using/profiles.adoc:
```diff
@@ -15,46 +15,52 @@ A profile is an instance of the Minishift VM along with all of its configuration
-You can determine the active profile by using the xref:../command-ref/minishift_profile_list.adoc#[`minishift profile list`] command. 
-You can execute single commands against a non-active profile, by using the `--profile` flag.
-Example:
+There is always a so called active profile.
```
"So called" has a negative connotation. In colloquial/conversational use, it carries the meaning of "says ______, but doesn't mean it." Let's avoid that connotation.

I think the original sentence ("the current profile being used is called the active profile") worked perfectly well.

kowen-rh commented on docs/source/using/profiles.adoc:
```diff
@@ -15,46 +15,52 @@ A profile is an instance of the Minishift VM along with all of its configuration
-You can run the `profile set` command for a non existing profile to set it as the active profile.
-The profile is automatically created when you subsequently run any `minishift` command.
-See xref:../command-ref/minishift_profile_set.adoc#[`minishift profile set`].
+NOTE: A profile becomes automatically the active profile when a {project} instance is started successfully via `minishift start`.
```
Please switch "becomes automatically" to "automatically becomes".

Also, personally, I prefer the use of blocks for admonitions -- this makes them stand out, and they become easier to edit in the future in case more information needs to be added:

```adoc
[NOTE]
====
{content}
====
```

hferentschik replied:
> Please switch "becomes automatically" to "automatically becomes".

+1

> Also, personally, I prefer the use of blocks for admonitions -- this makes them stand out, and they become easier to edit in the future in case more information needs to be added:

Same here. I used them as well, but at some stage someone switched to NOTE:

kowen-rh commented on docs/source/using/profiles.adoc:
```diff
@@ -63,75 +69,66 @@ The default profile is *minishift*.
+To create a new profile and configure its the xref:../using/basic-usage.adoc#persistent-configuration[persistent configuration] you have two options.
+The first option is to implicitly create the new profile by making it the active profile using the `profile set` command.
+Once the profile is active you can run any `minishift config` command.
+Lastly start the instance:
```
Comma: "Lastly, start the instance:"

kowen-rh reviewed (changes requested):
+1 for giving profiles higher priority in the topic map and for showing the output of commands.

Most of the changes I can quickly see are very minor, but important.

Thanks!

hferentschik commented:
> +1 for giving profiles higher priority in the topic map and for showing the output of commands.

thanks

hferentschik committed:
Issue #1485 Refining profile documentation

- Adding output to the exmaple commands
- Expanding on the example workflow
- Clarify wording around active profile
- Making docs more compact by removing unnecessary fillers

LalatenduMohanty commented:
Merged, Thanks.
